test/onnx/test_operators.py

```diff
@@ -877,6 +879,41 @@ def test_det(self):
         x = torch.randn(2, 3, 5, 5, device=torch.device('cpu'))
         self.assertONNX(lambda x: torch.det(x), x, opset_version=11)
 
+    @unittest.skip("disable test until onnx submodule is updated")
```

Comment:
Shall we enable the tests in test_opeartor.py by setting enable_onnx_checker to False?

Reply:
The `assertONNX` also runs onnx_checker.

Reply:
Shall we add a flag to disable that as well in assertONNX?

Reply:
Or we should remove the expect files :-)

Reply:
@houseroad there is no clean way to just disable onnx checker because export_to_pretty_string doesn't take enable_onnx_checker flag: https://github.com/pytorch/pytorch/blob/1d8012a624e4dbc9f66c7942e82e168707796855/torch/onnx/utils.py#L412
As of now it's only possible to disable onnx cheker for all tests or none.
Do you want expect files removed?